Makefile

```diff
@@ -161,7 +166,7 @@ $(GOPATH)/bin/go-bindata:
 cross: out/localkube out/minikube-linux-amd64 out/minikube-darwin-amd64 out/minikube-windows-amd64.exe
 
 .PHONY: cross-e2e
```

Comment:
This looks like it might have been a typo rom before, but the phony is cross-e2e and the target is e2e-cross